client/src/components/LoginForm/SignupContent.jsx

```diff
@@ -3,20 +3,18 @@ import Section from '../Section';
 import EditText from '../EditText';
 import DropDown from '../DropDownOutlined/DropDown';
 
-const years = ['2014', '2015', '2016', '2017', '2018', '2019'];
-const majors = [
-  'Computer Engineering',
-  'Electrical Engineering',
-  'Software Engineering',
-];
+const tempYears = ['2014', '2015', '2016', '2017', '2018', '2019'];
+const tempCurriculum = ['7 semesters', '8 semesters'];
+const tempMajors = ['Computer', 'Electrical', 'Software'];
```

Comment:
The names would have to be exactly
'Computer Engineering',
  'Electrical Engineering',
  'Software Engineering' if you want to hardcode it, but it would probably be better to grab that at the same time as the years now that I think of it. 
The curriculum would also have to be exactly "7-semester-curriculum" or "8-semester-curriculum". But also, should be grabbed from backend. I can change the endpoint for you.

Reply:
I'll change the fields when sending it in the request, but we don't want to display `7-semester-curriculum` for the user. They don't need the dashed ☺️